### PR TITLE
[EFR-02]Enterprise Feature Request - False Positive Triaging

### PR DESCRIPTION
**Pull request by ajinabraham**
### Describe the Pull Request

```
Enterprise Feature Request - False positive suppression support
```
Coverage

- [x] Android - Code, Manifest
- [x] iOS - Code 
- [x] iOS Binary
- [x] Support other params that contribute to score
- [x] API + Docs
- [x] Support for Score/PDF/JSON etc.
- [ ] Tests

[Fixes](https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/2000/commits/bbf0c0d03ea45e5ee3f1f3f0f972b68416a38580) https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/2043

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [ ] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [ ] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

File: mobsf/static/others/js/suppression.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    rfiles.forEach(element => {
+      html += `<li>${escapeHtml(element)}</li>`
+    });
+    html += '</div></div></br>'
+  }
+  return html
+}
+
+function list_suppressions(){
+  action(document.location.origin + '/list_suppressions/', { checksum: hash }, function(json) {
```
Comment (matandobr) [inline mobsf/static/others/js/suppression.js:136]: Don't you prefer using url names? this is prone to some errors I guess

**Review by matandobr — CHANGES_REQUESTED** [review]
**
image
**
This happened after I deleted a suppression by rule ID

Also I think the search is a bit weird because you are not splitting the lines, so each search will eaither return 0, 1, or 2 rows... not so helping I guess
image

File: mobsf/MobSF/settings.py [hunk 2]
```diff
@@ -149,7 +149,7 @@
 # End Postgres support
 """
 # ===============================================
-
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/MobSF/settings.py:152]: This is something needed in newer version of Django to support auto incrementing primary key in new modals

https://www.geeksforgeeks.org/autofield-django-models/

File: mobsf/static/others/js/suppression.js (hunk 1 above)
Comment (ajinabraham) [reply mobsf/static/others/js/suppression.js:136]: I can update that.

File: mobsf/MobSF/init.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 logger = logging.getLogger(__name__)
 
-VERSION = '3.5.2'
+VERSION = '3.6.0'
 BANNER = """
   __  __       _    ____  _____       _____  ____  
  |  \/  | ___ | |__/ ___||  ___|_   _|___ / | ___|
```
[the remark this answers is not in the document]
Comment (ajinabraham) [reply mobsf/MobSF/init.py:16]: Yea, sure

**Comment by ajinabraham** [comment]
@matandobr Addressed some of the review comments.

> This happened after I deleted a suppression by rule ID

Can you reproduce this reliably/ share steps to reproduce? I cannot reproduce this at my end.
I know that there is an edge case where Data tables will fail if the table is not created properly in time.


> Also I think the search is a bit weird because you are not splitting the lines, so each search will either return 0, 1, or 2 rows... not so helping I guess

I am not sure I understand you. The search is a feature provided by Data tables itself.

**Review by matandobr — APPROVED** [review]
LGTM, great work 🎈🤙

**Pull request merged**